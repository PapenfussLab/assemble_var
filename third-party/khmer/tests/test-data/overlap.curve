6021 0
12283 0
18155 0
24050 0
29649 40
35878 63
42077 63
48233 63
54423 63
60530 121
66015 154
71548 154
77733 154
83781 154
89819 154
95916 154
101765 168
107569 168
112958 168
119116 168
125245 168
131415 168
137387 293
142771 293
148585 293
154968 293
161049 295
166997 295
173045 295
178307 295
184486 295
190476 295
196578 295
202648 338
208367 351
213865 420
220200 420
226356 449
232545 449
238563 463
244631 463
250477 463
255751 463
261733 463
267940 468
274028 468
280168 468
286339 468
292468 468
297959 468
304094 468
310195 468
316381 468
322511 468
328748 468
334296 468
340307 468
346479 468
352467 468
358167 468
364100 468
369953 500
376021 500
382062 545
388117 545
393750 545
398447 1430
401544 4527
404607 7590
408026 11009
413425 16408
419262 22245
424872 27855
430628 33611
436424 39407
442091 45074
447704 50687
453532 56515
459256 62239
465319 68302
471277 74260
477238 80221
483201 86184
488559 91542
494427 97410
500543 103526
506488 109471
512256 115239
518250 121233
524122 127105
529993 132976
535602 138585
541291 144274
547407 150390
553356 156339
559303 162286
565185 168168
570992 173975
576607 179590
581866 184849
6021 0
12283 0
18155 0
24050 0
29649 40
35878 63
42077 63
48233 63
54423 63
60530 121
66015 154
71548 154
77733 154
83781 154
89819 154
95916 154
101765 168
107569 168
112958 168
119116 168
125245 168
131415 168
137387 293
142771 293
148585 293
154968 293
161049 295
166997 295
173045 295
178307 295
184486 295
190476 295
196578 295
202648 338
208367 351
213865 420
220200 420
226356 449
232545 449
238563 463
244631 463
250477 463
255751 463
261733 463
267940 468
274028 468
280168 468
286339 468
292468 468
297959 468
304094 468
310195 468
316381 468
322511 468
328748 468
334296 468
340307 468
346479 468
352467 468
358167 468
364100 468
369953 500
376021 500
382062 545
388117 545
393750 545
398447 1430
401544 4527
404607 7590
408026 11009
413425 16408
419262 22245
424872 27855
430628 33611
436424 39407
442091 45074
447704 50687
453532 56515
459256 62239
465319 68302
471277 74260
477238 80221
483201 86184
488559 91542
494427 97410
500543 103526
506488 109471
512256 115239
518250 121233
524122 127105
529993 132976
535602 138585
541291 144274
547407 150390
553356 156339
559303 162286
565185 168168
570992 173975
576607 179590
581866 184849
