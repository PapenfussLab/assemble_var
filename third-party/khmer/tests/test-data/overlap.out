/u/qingpeng/Github/newkhmer/tests/test-data/test-overlap1.fa:
# of unique kmers:440346
# of occupied bin:440299
/u/qingpeng/Github/newkhmer/tests/test-data/test-overlap2.fa:
# of unique kmers:581866
# of occupied bin:581783
# of overlap unique kmers:184849
